roles/setup_hammerdb/templates/loader.tcl.template

```diff
@@ -11,6 +11,12 @@ diset connection pg_port {{ pg_port }}
 diset tpcc pg_count_ware {{ tprocc_warehouses }}
 diset tpcc pg_num_vu {{ ansible_processor_vcpus }}
 diset tpcc pg_raiseerror true
-diset tpcc pg_superuser {{ pg_superuser }}
+{% if azure_db_hackery %}
```

Comment:
This variable does not look to be defined in `setup_hammerdb/defaults/main.yml`

Reply:
Right, it's set on the fly when a `deploy` is executed: https://github.com/mw2q/postgres-deployment/blob/azure-db/edbdeploy/project.py#L1261